odoo/modules/loading.py

```diff
@@ -416,6 +416,10 @@ def load_modules(db, force_demo=False, status=None, update_module=False):
         for kind in ('init', 'demo', 'update'):
             tools.config[kind] = {}
 
+        for vals in registry._notnull_errors.values():
+            msg, tablename, columnname, query, exception = vals
+            _logger.warning(msg, tablename, columnname, query, exc_info=exception)
```

Comment:
I would move this at the same position as the call to `registry.finalize_constraints()` from saas13.3 on, because
- `finalize_constraints()` serves a very similar purpose, and
- this bug fix will need to be adapted anyway for `finalize_constraints()`.